src/containers/AboutIndex/components/MembersCard.tsx

```diff
@@ -0,0 +1,44 @@
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Paper from 'components/layout/Paper';
+import PeopleIcon from '@material-ui/icons/People';
+import PersonIcon from '@material-ui/icons/Person';
+export type MembersCardProps = {
+  members: string[];
+};
+
+const useStyles = makeStyles((theme) => ({
+  icons: {
+    marginRight: theme.spacing(1),
+  },
+}));
+
+const MembersCard = ({ members }: MembersCardProps) => {
+  const classes = useStyles();
+  return (
+    <Paper>
+      <Grid container spacing={2}>
+        <Grid item xs={12}>
+          <Box alignItems='center' display='flex' flexWrap='wrap'>
+            <PeopleIcon className={classes.icons} fontSize='large' />
+            <Typography variant='h2'>Medlemmer</Typography>
+          </Box>
+        </Grid>
+        {members.map((member: string) => {
```

Comment:
Bør ikke være nødvendig å si at `member` er en string her, Typescript skjønner det selv
```suggestion
        {members.map((member) => {
```